### PR TITLE
Add mapping for multilingual subject without lang attributes

### PR DESCRIPTION
**Pull request by jcoyne**
**NOTE:  Person merging should ensure that any mapping changes are ticketed as a cocina mapping in dor-services-app, e.g. dor-services-app/issues/1251**

## Why was this change made?

we came across this XML in the repo: https://github.com/sul-dlss/dor-services-app/issues/1860

### CODE REVIEW

**Review by arcadiafalcone — APPROVED** [review]

File: mods_cocina_mappings/mods_to_cocina_subject.txt [hunk 1]
```diff
@@ -715,6 +715,33 @@ Based on jw325wd1971
...
+</subject>
+<subject altRepGroup="1">
+  <cartographics>
+    <scale>&#x6BD4;&#x4F8B;&#x5C3A; 1:650,000.</scale>
+  </cartographics>
+</subject>
+
+{
+  "form": [
+    {
```
Comment (jcoyne) [inline mods_cocina_mappings/mods_to_cocina_subject.txt]: should this be a parallelValue?

**Review by jcoyne — COMMENTED** [review]
This seems to be in conflict with https://github.com/sul-dlss/dor-services-app/commit/c25ffb59a71b21c5647b2c5b7b5049212707188d. @justinlittman

File: mods_cocina_mappings/mods_to_cocina_subject.txt (hunk 1 above)
Comment (arcadiafalcone) [reply mods_cocina_mappings/mods_to_cocina_subject.txt]: Yes, it should.

**Comment by justinlittman** [comment]
What is the conflict?

**Comment by jcoyne** [comment]
Doesn't that prevent having only a scale?

**Comment by justinlittman** [comment]
Entirely possible I'm wrong, but I think that is testing the subject side, not the form side.

https://github.com/sul-dlss/dor-services-app/commit/c25ffb59a71b21c5647b2c5b7b5049212707188d#diff-13bbee3fbb99afe7f2834cc6558980b438d8300e382c49187755bf32aaf9c8f2R614

**Review by justinlittman — APPROVED** [review]

**Pull request merged**